features/exacte_match.feature

```diff
@@ -0,0 +1,147 @@
+  En exacteMatch=false
+  Dan bevat het resultaat alle 3 de objecten
+	
+  Gegeven een verzameling van 3 objecten met postcode=1234AA 
+  En één object (1) met huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging
+  En één object (2) met huisnummer=3
+  En huisletter=A
+  En geen huisnummertoevoeging
+  En één object (3) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  Als er wordt gezocht met postcode=1234AA 
+  En huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging 
+  En exacteMatch=true
+  Dan bevat het resultaat alleen het object (1) met postcode=1234AA 
+  En huisnummer=3
+
+Scenario: combinatie van huisnummer en huisnummertoevoeging
+  Gegeven een verzameling van 5 objecten met postcode=1234AA 
+  En één object (1) met huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging
+  En één object (2) met huisnummer=3
+  En huisletter=A
+  En geen huisnummertoevoeging
+  En één object (3) met huisnummer=3
+  En huisletter=A
+  En huisnummertoevoeging=bis
+  En één object (4) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  En één object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3 
+  En huisnummertoevoeging=bis
+  En geen huisletter 
+  En exacteMatch=false
+  Dan bevat het resultaat object (3) met huisnummer=3
+  En huisletter=A 
+  En huisnummertoevoeging=bis 
+  En object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+	
+  Gegeven een verzameling van 5 objecten met postcode=1234AA 
+  En één object (1) met huisnummer=3
+  En geen huisletter
+  En geen huisnummertoevoeging
+  En één object (2) met huisnummer=3
+  En huisletter=A
+  En geen huisnummertoevoeging
+  En één object (3) met huisnummer=3
+  En huisletter=A
+  En huisnummertoevoeging=bis
+  En één object (4) met huisnummer=3
+  En huisletter=B
+  En geen huisnummertoevoeging
+  En één object (5) met huisnummer=3
+  En huisletter=B
+  En huisnummertoevoeging=bis
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3 
+  En huisnummertoevoeging=bis
+  En geen huisletter 
+  En exacteMatch=true
+  Dan bevat het resultaat geen objecten
+
+Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor en exacteMatch wordt niet toegepast
+  Gegeven een verzameling van 2 objecten met postcode=1234AA
+  En huisnummer=3 
+  En geen huisletter 
+  En geen huisnummertoevoeging
+  Als er wordt gezocht met postcode=1234AA
+  En huisnummer=3
+  En geen huisnummertoevoeging
+  En geen huisletter 
+  En exacteMatch=false
+  Dan bevat het resultaat beide objecten
+  
+Scenario: combinatie van huisnummer, huisletter en/of huisnummertoevoeging komt meerdere keren voor en exacteMatch wordt toegepast
+  Gegeven een verzameling van 2 objecten met postcode=1234AA
+  En huisnummer=3 
+  En geen huisletter 
+  En geen huisnummertoevoeging
```

Comment:
Als je een verzameling hebt van 2 objecten die een postcode en huisnummer delen dan moet er toch op zijn minst op een van die objecten nog een huisletter aanwezig zijn?

Reply:
@melsk-r , Officieel mogen er geen identieke postcode huisnummer combinaties zijn. Hier wordt ook op gerapporteerd. In de praktijk kan een postcode huisnummer combinatie meerdere keren voorkomen ook zonder huisletter en huisnummertoevoeging. 
Bijvoorbeeld als een woning wordt afgebroken en op hetzelfde perceel nieuwbouw plaatsvindt komt dat voor.

Reply:
Ok duidelijk, is dat ook de situatie die jullie hier willen simuleren?

Reply:
ja